common/metric_types.py

```
@@ -288,6 +298,38 @@ async def _send_request(
+    @staticmethod
+    def get_params_from_state(state_data: dict[str, Any]) -> list[Any]:
+        """Return empty params list for eth_blockNumber (JSON-RPC positional params)."""
+        return []
+
+    def _on_json_response(self, json_response: dict[str, Any]) -> None:
+        result = json_response.get("result")
+        if isinstance(result, str):
+            try:
+                self._captured_block_number = int(result, 16)
+            except ValueError:
+                pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(ValueError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(ValueError): ...`

(SIM105)